vite-project/src/components/Navbar.jsx

```diff
@@ -0,0 +1,20 @@
+import { Component } from 'react';
+import './Navbar.css';
+
+class Navbar extends Component{
```

Comment:
Class Components of course work, and you will see them in production code bases, but the industry standard is now to use functional components.

Here is a guide on how to migrate across:
https://react.dev/reference/react/Component#alternatives

Using functional components allows for (arguably) easier state management through the use of hooks like useState